prometheus_client/exposition.py

```diff
@@ -97,10 +97,10 @@ def log_message(self, format, *args):
 
 
 def start_http_server(port, addr=''):
-    """Spawns each HTTPServer in a new thread to prevent blocking"""
+    """Spawns each HTTPServer in a new thread to prevent blocking."""
     class ThreadingSimpleServer(ThreadingMixIn, HTTPServer):
         pass
-    """Starts a HTTP server for prometheus metrics in a new thread"""
+    """Starts a HTTP server for prometheus metrics as a daemon thread."""
```

Comment:
The daemon thread should be mentioned in start_http_server, as that's the primary docs the user will see.